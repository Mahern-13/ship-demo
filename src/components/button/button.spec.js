import React from "react";
import { shallow } from "enzyme";
import { Default, Primary, Warning, Danger } from "./index";

describe("Default Button", () => {
  it("renders the component", () => {
    const wrapper = shallow(<Default text="test" />);
    expect(wrapper).toMatchSnapshot();
  });

  it("renders the component with disabled prop", () => {
    const wrapper = shallow(<Default text="test" disabled={true} />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("Primary Button", () => {
  it("renders the component", () => {
    const wrapper = shallow(<Primary text="test" />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("Warning Button", () => {
  it("renders the component", () => {
    const wrapper = shallow(<Warning text="test" />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("Danger Button", () => {
  it("renders the component", () => {
    const wrapper = shallow(<Danger text="test" />);
    expect(wrapper).toMatchSnapshot();
  });
});
