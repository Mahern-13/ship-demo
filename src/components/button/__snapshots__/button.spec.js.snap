// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Danger Button renders the component 1`] = `ShallowWrapper {}`;

exports[`Default Button renders the component 1`] = `ShallowWrapper {}`;

exports[`Default Button renders the component with disabled prop 1`] = `ShallowWrapper {}`;

exports[`Primary Button renders the component 1`] = `ShallowWrapper {}`;

exports[`Warning Button renders the component 1`] = `ShallowWrapper {}`;
